common/src/main/resources/less/gpp-grid.less

```diff
@@ -0,0 +1,91 @@
+@two-column-break: 60em;
+@three-column-break: 90em;
+
+.gpp-grid {
+    display: grid;
+    grid-gap: 1em;
+
+    /*
+     You do not not to specify the row and column of each cell 
+     for most simple layouts, such as most forms. It will fill 
+     in each cell in a row and then go to the next column.
+    */
```

Comment:
If you like this concept, I can add classes for specifying rows and columns, as well as row and column spans. This would be useful for more complex layouts.